hack/render-manifests.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"text/template"
+
+	"github.com/pkg/errors"
+)
+
+func exitWithError(err error, cause string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, "render-manifests.go: error: %v\n", errors.Wrapf(err, cause, args))
+	os.Exit(1)
+}
+
+func main() {
+	type Inventory struct {
+		Namespace           string
+		NMStateHandlerImage string
+		ImagePullPolicy     string
+	}
+
+	inventory := Inventory{
+		Namespace:           os.Args[1],
+		NMStateHandlerImage: os.Args[2],
+		ImagePullPolicy:     os.Args[3],
+	}
+	inputDir := os.Args[4]
+	outputDir := os.Args[5]
+
+	// Clean up output dir so we don't have old files.
+	err := os.RemoveAll(outputDir)
+	if err != nil {
+		exitWithError(err, "failed cleaning up output dir %s", outputDir)
+	}
+
+	tmpl, err := template.ParseGlob(path.Join(inputDir, "*.yaml"))
+	if err != nil {
+		exitWithError(err, "failed parsing top dir manifests at %s", inputDir)
+	}
+
+	tmpl, err = tmpl.ParseGlob(path.Join(inputDir, "*/*.yaml"))
```

Comment:
please use better variables  for tmpl, I don't understand why you override it now.

Reply:
@qinqon

Reply:
I see now. Thanks